packages/dnb-design-system-portal/src/docs/uilib/components/dropdown/Examples.tsx

```diff
@@ -287,21 +288,29 @@ export const DropdownIconLeft = () => (
 
 export const DropdownActionMenu = () => (
   <Wrapper>
-    <ComponentBox scope={{ data }} data-visual-test="dropdown-action_menu">
+    <ComponentBox
+      scope={{ trash, download }}
+      data-visual-test="dropdown-action_menu"
+    >
       <Dropdown
         title="ActionMenu"
         action_menu={true}
         align_dropdown="left"
-        data={[
-          <>
-            <IconPrimary icon="check" right />
-            Save
-          </>,
-          <>
-            <IconPrimary icon="download" right />
-            Download
-          </>,
-        ]}
+        data={() => ({
+          trash: (
+            <>
+              <Icon icon={trash} right />
+              Save
```

Comment:
How it looks:
<img width="275" alt="image" src="https://github.com/dnbexperience/eufemia/assets/1359205/a38c7b78-9711-4918-848e-da89d2627088">

```suggestion
              Move to trash
```

I know it's just an example, but feels a bit strange to have a trash icon together with the text "Save" :P
Hence, I'm suggesting to change the text to "Move to trash" or so 😸